README.md

```diff
@@ -84,30 +84,68 @@ DMeta is an open source Python package that provides ... TODO
 - Run `pip install .`
 
 ## Usage
+### Programmatically 
```

Comment:
I suggest using `### In Python` instead.

Reply:
Go ahead and apply yourself, then finally ping me, I will review them.